AdminWebsite/AdminWebsite.UnitTests/Services/HearingServiceTests.cs

```diff
@@ -969,6 +1004,28 @@ public async Task EditHearingSendConfirmation_Should_Receive_Call_When_HearingTy
                 Times.Exactly(3));
         }
 
+        [Test]
+        public async Task EditHearingSendConfirmation_Judge_Should_not_Receive_Call_Without_judge_email_address_when_HearingType_Is_DEMO_for_Confirmedhearing()
```

Comment:
I know the previous test was already here but is the name Should_Receive_Call valid?